ibm/data_source_ibm_is_security_group_target.go

```diff
@@ -0,0 +1,91 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceIBMISSecurityGroupTarget() *schema.Resource {
+	return &schema.Resource{
+
+		Read: dataSourceIBMISSecurityGroupTargetRead,
+
+		Schema: map[string]*schema.Schema{
+
+			"security_group_id": {
```

Comment:
rename security_group_id to security_group in VPC we are not using any _id ..We will follow same here also